amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
how sensitive is this to the state of the test db, and will this be heavily impacted when we reapply the scrub method again in the future?